docs/lib/datastore/fragments/ios/getting-started/20_installLib.md

```diff
@@ -0,0 +1,22 @@
+
+To install the Amplify Storage and Authentication to your application, add `AmplifyPlugins/AWSDataStorePlugin`. Your `Podfile` should look similar to:
```

Comment:
again, doesn't look like we are adding authentication
it should say, 
To install the Amplify Datastore plugin to your application,